Add pre-fill for the command handler tests

oechsler opened the pull request.

oechsler commented:
Do we need translation stuff here?

msvujnovic-becklyn commented on src/Resources/skeleton/ddd/command-handler-test.tpl.php:
```diff
@@ -31,9 +33,31 @@ protected function setUp () : void
         $this->commandHandlerPostSetUp();
     }
 
+    public function test_TESTED_LOGIC_Is_ACTIONED_And<?= $entity; ?>IsDequeuedByEventRegistry() : void
```
`_TESTED_LOGIC_Is_ACTIONED_` -> `_TESTED_LOGIC_IS_ACTIONED_`

Translation for 'And' & 'IsDequeuedByEventRegisty' is needed

msvujnovic-becklyn commented on src/Resources/skeleton/ddd/command-handler-test.tpl.php:
```diff
@@ -31,9 +33,31 @@ protected function setUp () : void
+    {
+        $<?= $strtocamel($entity); ?>Id = $this->given<?= $entity; ?>Id();
+        // TODO add additional command params
+        $<?= $strtocamel($entity); ?> = $this->given<?= $entity; ?>CanBeFoundById($<?= $strtocamel($entity); ?>Id);
```
Translation for 'given' and 'CanBeFoundById' is needed

msvujnovic-becklyn commented on src/Resources/skeleton/ddd/command-handler-test.tpl.php:
```diff
@@ -31,9 +33,31 @@ protected function setUp () : void
+        $<?= $strtocamel($entity); ?>Id = $this->given<?= $entity; ?>Id();
+        // TODO add additional command params
+        $<?= $strtocamel($entity); ?> = $this->given<?= $entity; ?>CanBeFoundById($<?= $strtocamel($entity); ?>Id);
+        $this->then_SOMETHING_ShouldBe_ACTIONED_($<?= $strtocamel($entity); ?> /** TODO additional params **/);
```
Translation for 'then' and 'ShouldBe' is needed

msvujnovic-becklyn commented on src/Resources/skeleton/ddd/command-handler-test.tpl.php:
```diff
@@ -31,9 +33,31 @@ protected function setUp () : void
+        // TODO add additional command params
+        $<?= $strtocamel($entity); ?> = $this->given<?= $entity; ?>CanBeFoundById($<?= $strtocamel($entity); ?>Id);
+        $this->then_SOMETHING_ShouldBe_ACTIONED_($<?= $strtocamel($entity); ?> /** TODO additional params **/);
+        $this->then<?= $entity; ?>ShouldBeDequeuedByEventRegistry($<?= $strtocamel($entity); ?>->reveal());
```
Translation for 'then' and 'ShouldBeDequeuedByEventRegistry' is needed

msvujnovic-becklyn commented on src/Resources/skeleton/ddd/command-handler-test.tpl.php:
```diff
@@ -31,9 +33,31 @@ protected function setUp () : void
+        $<?= $strtocamel($entity); ?> = $this->given<?= $entity; ?>CanBeFoundById($<?= $strtocamel($entity); ?>Id);
+        $this->then_SOMETHING_ShouldBe_ACTIONED_($<?= $strtocamel($entity); ?> /** TODO additional params **/);
+        $this->then<?= $entity; ?>ShouldBeDequeuedByEventRegistry($<?= $strtocamel($entity); ?>->reveal());
+        $this->when<?= $extra["command_namespace"]; ?>IsHandled($<?= $strtocamel($entity); ?>Id /** TODO additional command params **/);
```
Translation for 'when' and 'IsHandled' is needed

msvujnovic-becklyn commented on src/Resources/skeleton/ddd/command-handler-test.tpl.php:
```diff
@@ -31,9 +33,31 @@ protected function setUp () : void
+        $this->when<?= $extra["command_namespace"]; ?>IsHandled($<?= $strtocamel($entity); ?>Id /** TODO additional command params **/);
+    }
+
+    private function then_SOMETHING_ShouldBe_ACTIONED_(ObjectProphecy | <?= $entity; ?> $<?= $strtocamel($entity); ?> /** TODO additional params **/) : void
```
Translation for 'then' and 'ShouldBeDequeuedByEventRegistry' is needed

msvujnovic-becklyn commented on src/Resources/skeleton/ddd/command-handler-test.tpl.php:
```diff
@@ -31,9 +33,31 @@ protected function setUp () : void
+        $<?= $strtocamel($entity); ?>->_DO_ACTION_(/** TODO add params **/)->shouldBeCalled();
+    }
+
+    private function when<?= $extra["command_namespace"]; ?>IsHandled(<?= $entity; ?>Id $<?= $strtocamel($entity); ?>Id /** TODO additional command params **/) : void
```
Translation for 'when' and 'IsHandled' is needed

msvujnovic-becklyn commented on src/Resources/skeleton/ddd/entity-test-trait.tpl.php:
```diff
@@ -63,4 +63,9 @@ protected function <?= $i18n["test"]["_given"]; ?><?= $entity; ?><?= $i18n["test
         $this-><?= $strtocamel($entity); ?>Repository->findOneById($<?= $strtocamel($entity); ?>Id)->willThrow(new <?= $entity; ?><?= $i18n["not_found"]; ?>Exception());
     }
+
+    protected function then<?= $entity; ?>ShouldBeDequeuedByEventRegistry($<?= $strtocamel($entity); ?>) : void
```
Translation for 'then' and 'ShouldBeDequeuedByEventRegistry' is needed

msvujnovic-becklyn commented:
Yep, I noted all the places I could see which need translation. Some of the parts which need translation already have it implemented, others you will need to create.

glassp reviewed (approved):
Should be good after the changes Marko requested

Commit:
Add translations to command handler test

then_SOMETHING_ShouldBe_ACTIONED_ method now has translatitons for english and german

Commit:
Add translations to command handler test

whenCommandIsHandled method now has translatitons for english and german

Commit:
Fix syntax error

Commit:
Add suffix for german translation

Now ShouldBeDequeuedByEventRegistry is translated to a more fitting german sentence

msvujnovic-becklyn commented on src/Resources/i18n/de.yaml:
```diff
@@ -15,16 +15,21 @@ test:
     by_id: DurchId
     can_be_found_by_id: KannDurchIdGefundenWerden
     cannot_be_found_by_id: KannDurchIdNichtGefundenWerden
+    dequeued_by_event_registry: VonDerEventRegisitrierungAusDerWarteschlangeEntfernt
```
Bwah :D

Machma lieber `VonDerEventRegistryDequeued`, weil die event registry ist da auf Englich in der Codebasis, und wir benutzen auch Englische begriffe wenn wir drüber reden.

oechsler replied:
Ok, kann auch nix dafür, dass Deutsch dreck is 🤣

Commit:
Improve german translations

Translation of DequeuedByEventRegistry now uses a more informal but shorter translation

Commit:
Add pre-fill for the command handler tests

Commit:
Add translation for command handler test

TESTED_LOGIC_IS_ACTIONED method now has translations for en and de

Commit:
Add translations to entity test trait

ShouldBeDequeuedByEventRegistry method now has translations for english and german

Commit:
Switch to code style used by other templates
